app/indexers/plum_jsonld_converter.rb

```diff
@@ -144,7 +145,7 @@ def subject_label
 
     def subject_with_category
       JSON.generate(
-        json["subject"].map do |x|
+        json["subject"].select { |x| x["in_scheme"] }.map do |x|
```

Comment:
@tpendragon did this present as a bug in indexing? Does this mean there are records with subjects but not categories? If so I think the subjects won't be shown in the view any more; I removed them there in favor of the new presentation.

maybe we should consider allowing for category to be blank if this seems like an issue.